UI/TicketEditUI/TicketEditUI.cs

```diff
@@ -0,0 +1,121 @@
+﻿using Logic;
+using Model;
+using Model.Enums;
+
+namespace UI.TicketEditUI
+{
+    public partial class TicketEditUI : Form
+    {
+        public bool canceled = false;
+        private bool isEditing = false;
+        private Ticket _ticket;
+        public Ticket Ticket { get { return _ticket; } }
+        private TicketService _ticketService;
```

Comment:
Maybe you can put this locally at editTicketBtn_Click, because it's only used there.

Reply:
i changed some things idk it it was you meant can you check it